src/ol/format/format.js

```diff
@@ -5,6 +5,7 @@ goog.provide('ol.format.FormatType');
  * @enum {string}
  */
 ol.format.FormatType = {
+  ARRAY_BUFFER: 'arraybuffer',
```

Comment:
We had that and removed it. I think as long as ol.format.MVT is the only format that uses it, it makes no sense to have a base class. Just makes the code more complicated than it needs to be.